README.md

```diff
@@ -3,13 +3,22 @@ This is application is a simple tic tac toe game that is created utilizing ruby
 
 ## Starting the Game
 The game will launch through a runner file and will allow a human player to play against another human player and a computer with varying degrees of difficulty. The game will be launched in the console with the following command:
-    `ruby game.rb`
+    `ruby lib/runner.rb`
 
-## Starting testing
+## Playing the Game
+The game will launch and will display the tic tac toe board.
+
+ 1 | 2 | 3
+---+---+---
+ 4 | 5 | 6
+---+---+---
+ 7 | 8 | 9
+
+ Player One will be prompted to select a number between 1 and 9 that will mark an X on the board. Player two will then be prompted to do the same thing in the terminal to mark an O on the board. The game ends when a player has successfully mark three spaces in succession vertically, horizontally, or diagonally. If no player connects three spaces, then the game will end in a tie.
+
+ ## Starting testing
 For testing, `rpec` will used in the development process to test the desired behavior of the tic tac toe game. `rspec` will need to be launched a separate console.
     `bundle exec guard`
-This will launch rspec will update tests continously in real time that are created in the `game_spec.rb` file.
+This will launch rspec will update tests continuously in real time that are created in the `spec` folder. Once this is operating, simply press <kbd>return</kbd> to launch all the tests.
```

Comment:
adding on to the testing instructions, explaining how to run just a single test file instead of all the tests at once might be useful for developers. (if this is possible)

Reply:
add individual test in the console